source/ocsp-support/ocsp-support.rst

```diff
@@ -100,9 +100,12 @@ invalid, the driver SHOULD end the connection.
     attempt to validate the status of the unvalidated certificates
     using the cached CRLs.
 
-7.  If the server’s certificate remains unvalidated and that certificate
-    has an OCSP endpoint, the driver SHOULD reach out to the OCSP
-    endpoint specified and attempt to validate that certificate.
+7.  If the server's certificate remains unvalidated and that certificate
+    has a list of OCSP responder endpoints, the driver SHOULD send HTTP requests
+    to the responders in parallel. The first valid response that concretely marks
+    the certificate status as good or revoked should be used. A five-second timeout
+    MUST be used for the requests. The status for a response should only be checked
+    if there are no errors parsing it and it is correctly signed.
```

Comment:
As a side note, I think we should remove the word "intermediate" from point 10, line 121, since I believe even if the leaf cert's status is unknown, we'll still connect (i.e. soft fail)

Reply:
Done.

Reply:
Thanks!